dev/src/__tests__/navbarResponsive.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { Builder, By, Browser, IRectangle, Capabilities } from 'selenium-webdriver'
+import { isOverlapping } from './testHelpers'
+
+describe("Front page elements login and logo should not overlap", () => {
+    //Laptop, desktop, mobile
+    it("Should not overlap on res 1920x1080", async () => {
```

Comment:
If the table driven testing style suggested in `courseUtilityLinks` works, it can be used in a similar fashion for these tests aswell.